app/models/post.rb

```diff
@@ -1,3 +1,14 @@
 class Post < ApplicationRecord
   belongs_to :user, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Let's improve this association to be more descriptive and conventional. Posts belong to an `author`, not `user`. Now since there is no model called Author, specifying User is just about enough, and we may not really need to foreign key.

  ```
  belongs_to :author, class_name: 'User'
  ```